src/components/shared/addVersionModal/addVersionModal.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { createVersion } from '../../../services';
+import { Modal } from '../../modal/modal';
+import { Input } from '../../modal/input';
+
+export const AddVersionModal = ({ isOpen, onCancel }) => {
```

Comment:
Instead of having modals it will be better that you have Add and Edit screens and put them in modal. So if it will be needed to use out of modal (for example in new page) you'll  be able to do it quickly